CloudKit-demo/Classes/ViewController/SelectCityViewController.swift

```diff
@@ -51,30 +52,32 @@ class YALSelectCityViewController: YALBaseViewController, UITableViewDataSource,
     
     // MARK: UITableViewDataSource
     func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return YALCity.defaultContent().count
+        return City.defaultContent().count
     }
     
     func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCellWithIdentifier(kCellReuseId, forIndexPath: indexPath)
         
-        let cityName = YALCity.defaultContent()[indexPath.row]["name"]
+        let cityName = City.defaultContent()[indexPath.row]["name"]
         cell.textLabel?.text = cityName
         
+        cell.accessoryType = indexPath == selectedIndexPath ? .Checkmark : .None
+        
         return cell
     }
     
     // MARK: UITableViewDelegate
     func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
         let cell = tableView.cellForRowAtIndexPath(indexPath)
-        if cell?.accessoryType == .None {
-            cell?.accessoryType = .Checkmark
-        } else {
-            cell?.accessoryType = .None
-        }
+        
+        cell?.accessoryType = .Checkmark
+        selectedIndexPath = indexPath
     }
     
     func tableView(tableView: UITableView, didDeselectRowAtIndexPath indexPath: NSIndexPath) {
         let cell = tableView.cellForRowAtIndexPath(indexPath)
-        cell?.accessoryType = UITableViewCellAccessoryType.None
+        
+        cell?.accessoryType = .None
+        selectedIndexPath = indexPath
```

Comment:
@DarthRumata please check this line. Maybe you intended to set nil instead of indexPath. Thanks.